getting_started/getting_started.py

```diff
@@ -119,17 +121,17 @@ def lf_textblob_polarity(x):
 
 # %% [markdown]
 # And much more!
-# For many more types of labeling functions–including over data modalities beyond text–see the other [tutorials](#) and [real-world examples](#).
+# For many more types of labeling functions — including over data modalities beyond text — see the other [tutorials](#) and [real-world examples](#).
```

Comment:
Tutorials can point to the website tutorials page. If we don't have a link for real world examples, you can remove the link here.